src/core/crash-handler.c

```diff
@@ -102,6 +102,10 @@ _noreturn_ static void crash(int sig, siginfo_t *siginfo, void *context) {
                         siginfo_t status;
 
                         if (siginfo) {
+                                /* si_pid is not filled on SIGBUS. */
+                                if (sig == SIGBUS)
+                                        siginfo->si_pid = 1;
```

Comment:
hmm? this is wrong? `run0 bash -c 'echo $BASHPID && exec kill -BUS 1'` gives me correct pid. We need to conditionalize this.

Reply:
BTW, `si_pid` is properly set for me...